sphinx/tutorial/cylc/scheduling/integer-cycling.rst

```diff
@@ -412,7 +414,7 @@ starting 5 cycles *after* the initial cycle point.
       .. tip::
 
          You can get cylc graph to draw dotted boxes around the cycles by
-         clicking the "Organise by cycle point" button on the toolbar:
+         clicking the "organise by cycle point" button on the toolbar:
```

Comment:
This is the text on a label in the GUI (where it is spelt with a capital "O") - in quotes this does look wrong though. I have since discovered that sphinx provides some built-in roles which better handle things like this. In this case we should do:

```rst
clicking the :guilabel:`Organise by cycle point` button on the toolbar:
```
See the sphinx docs http://www.sphinx-doc.org/en/stable/markup/inline.html#role-guilabel

I have not yet started using all of these roles (they aren't all [properly] supported in the sphinx_rtd_theme) but the following are usefull:

* `guilabel`
* `menuselection`
* `term` - *already used with glossary definitions*

The tutorial does not [yet] use the first two, I intend to change this when I get the chance to make a second pass of the material.

Reply:
I agree actually that the capitalised version is more consistent and better - this was a change I was going to revoke upon review before submitting the PR but must have neglected, so I am glad you picked up on it. I did see a single 'guilabel' usage in the 'family triggers' tutorial which looked like a good option although only when used consistently, hence I commented on that file that the standalone use was confusing.

Reply:
Yes, this is something I only started doing with the further tutorials, we don't reference gui components often so there isn't much work to be done brining the rest of the material inline *[rst pun intended]*.